File: ProyectosCSharp/SoftProg/SoftProgModel/GestClientes/Cliente.cs
using SoftProgModel.RRHH;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftProgModel.GestClientes
{
    public class Cliente : Persona
    {
        private double lineaCredito;
        private Categoria categoria;

        public double LineaCredito { get => lineaCredito; set => lineaCredito = value; }
        public Categoria Categoria { get => categoria; set => categoria = value; }
    }
}

File: Arquitectura_REST_SOAP/Backend/SoftProgModel/src/main/java/pe/edu/pucp/softprog/gestclientes/model/Cliente.java
package pe.edu.pucp.softprog.gestclientes.model;
import java.util.Date;
import pe.edu.pucp.softprog.rrhh.model.Persona;
public class Cliente extends Persona{
    private double lineaCredito;
    private Categoria categoria;
    
    public Cliente(){}
    public Cliente(String dni, String nombre, String apellidoPaterno, char sexo, Date fechaNacimiento, double lineaCredito, Categoria categoria){
        super(dni,nombre,apellidoPaterno,sexo,fechaNacimiento);
        this.lineaCredito = lineaCredito;
        this.categoria = categoria;
    }

    public double getLineaCredito() {
        return lineaCredito;
    }

    public void setLineaCredito(double lineaCredito) {
        this.lineaCredito = lineaCredito;
    }

    public Categoria getCategoria() {
        return categoria;
    }

    public void setCategoria(Categoria categoria) {
        this.categoria = categoria;
    }
    
}
